src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/System.Text/Rune.cs

```diff
@@ -0,0 +1,1436 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Copied from https://github.com/dotnet/corefx/blob/e99ec129cfd594d53f4390bf97d1d736cff6f860/src/Common/src/CoreLib/System/Text/Rune.cs#L1
```

Comment:
>Copied [](start = 3, length = 6)

The source moved to dotnet/runtime:
[https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs)

Reply:
Fixed.  thanks.